packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -58,6 +71,27 @@ abstract class ProgressIndicator extends StatefulWidget {
   /// [ThemeData.accentColor].
   final Animation<Color> valueColor;
 
+  /// The [Semantics.label] for this progress indicator.
+  ///
+  /// This value indicates the purpose of the progress bar, and is used
+  /// by screen reading software to identify the widget.
+  final String semanticsLabel;
+
+  /// The [Semantics.value] for this progress indicator.
+  ///
+  /// This will be used in conjunction with the [semanticsLabel] by
+  /// screen reading software to identify the widget.
```

Comment:
This should probably say that its mostly use for determinate progress indicator to indicate how far along they are?

Reply:
See 42da5ff